### ISSUE TITLE
Cannot plot data if it all lies between integer exponents of log 10 (scale_*_log10)

### ISSUE THREAD

**Issue by r-barnes** [has2k1/plotnine#55]
I'm using plotnine v0.2.1 and attempting to make a log-log plot:

    import pandas as pd
    import matplotlib.pyplot as plt
    from plotnine import *

    df = pd.read_json("""{"index":{"0":49,"1":99,"2":199},"h":{"0":0.5,"1":0.25,"2":0.125},"t":{"0":25.0,"1":25.0,"2":25.0},"vals":{"0":[-60.7488535913,-3.2056655669],"1":[-47.4324189895,-2.3908288923],"2":[-35.7413627393,-1.8986336166]},"x":{"0":-60.7488535913,"1":-47.4324189895,"2":-35.7413627393},"y":{"0":-3.2056655669,"1":-2.3908288923,"2":-1.8986336166},"mag":{"0":60.8333749219,"1":47.4926355764,"2":35.7917563145}}""")

    #Works
    ggplot(df, aes(x='h', y='mag')) + geom_line()

    #Doesn't work
    ggplot(df, aes(x='h', y='mag')) + geom_line() + scale_y_log10() + scale_y_log10()

    #Works
    plt.loglog(df['h'],df['mag'])
    plt.show()

However, it fails with the error:

    ... ggplot(df, aes(x='h', y='mag')) + geom_line() + scale_y_log10() + scale_y_log10()
    Traceback (most recent call last):
      File "<stdin>", line 2, in <module>
      File "/usr/local/lib/python3.5/dist-packages/plotnine/ggplot.py", line 84, in __repr__
        self.draw()
      File "/usr/local/lib/python3.5/dist-packages/plotnine/ggplot.py", line 141, in draw
        self._build()
      File "/usr/local/lib/python3.5/dist-packages/plotnine/ggplot.py", line 264, in _build
        layout.setup_panel_params(self.coordinates)
      File "/usr/local/lib/python3.5/dist-packages/plotnine/facets/layout.py", line 183, in setup_panel_params
        self.panel_scales_y[j])
      File "/usr/local/lib/python3.5/dist-packages/plotnine/coords/coord_cartesian.py", line 68, in setup_panel_params
        out.update(train(scale_y, self.limits.xlim, 'y'))
      File "/usr/local/lib/python3.5/dist-packages/plotnine/coords/coord_cartesian.py", line 57, in train
        out = scale.break_info(rangee)
      File "/usr/local/lib/python3.5/dist-packages/plotnine/scales/scale.py", line 535, in break_info
        labels = self.get_labels(major)
      File "/usr/local/lib/python3.5/dist-packages/plotnine/scales/scale.py", line 629, in get_labels
        labels = self.trans.format(breaks)
      File "/usr/local/lib/python3.5/dist-packages/mizani/formatters.py", line 378, in _log_format
        dmin = np.log(np.min(x))/np.log(base)
      File "/usr/lib/python3/dist-packages/numpy/core/fromnumeric.py", line 2352, in amin
        out=out, **kwargs)
      File "/usr/lib/python3/dist-packages/numpy/core/_methods.py", line 29, in _amin
        return umr_minimum(a, axis, None, out, keepdims)
    ValueError: zero-size array to reduction operation minimum which has no identity

I think all of the values used in the loglog plot should be legal and, indeed, loglog plotting works in pyplot (I mention this not to lord pyplot over you better, but to try to rule out data issues).

**Comment by has2k1** [comment]
Minimal Reprex

```python
import pandas as pd
from plotnine import *

df = pd.DataFrame({'x': [0.5, .25, .125],
                   'y': [60, 50, 35]})

ggplot(df, aes('x', 'y')) + geom_line() + scale_y_log10()
```

Problem
Of the breaks created, none are in the [~35, ~60] interval. So they are stripped out; leaving 0 breaks.

**Comment by has2k1** [comment]
@r-barnes, is it possible for you to try out the solution

```
pip install git+https://github.com/has2k1/mizani.git@dev
```

Thanks.

**Comment by r-barnes** [comment]
Just tried it and everything's working for me now.

Thanks!

**Comment by has2k1** [comment]
I have released mizani v0.3.4 with the fix.
